Assets/DamageController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DamageController : MonoBehaviour {
+
+	void OnTriggerEnter(Collider other)
+	{
+		if (other.CompareTag ("Player")) {
```

Comment:
Wouldn't it be better to make the amount of health increase/decrease a public variable so that it is possible to changeit in the inspector?